assets/javascripts/discourse/components/share-report.hbs

```diff
@@ -0,0 +1,19 @@
+<div role="button" onclick={{action "open"}}>
+  {{d-icon "link"}}
+  {{group}}
+</div>
+
+{{#if visible}}
+  <div class="popup">
+    <label>{{i18n "explorer.link"}} {{group}}</label>
+    <input type="text" value={{link}}>
+
+    {{d-button
+      action="close"
```

Comment:
Pretty sure this'll print a deprecation notice? (if not let me know - we need to fix core)

We should it as a closure action

```suggestion
      action=this.close
```

Reply:
Oooh I'm confused because there are two `share-report.hbs` templates. The other one looks correct (but should be moved to make it colocated)